src/core/error_helpers.js

```diff
@@ -159,26 +160,53 @@ if (typeof IS_MINIFIED !== 'undefined') {
   // validateParameters() helper functions:
   // lookupParamDoc() for querying data.json
   var lookupParamDoc = function(func) {
-    var queryResult = arrDoc.classitems.filter(function(x) {
-      return x.name === func;
-    });
+    // look for the docs in the `data.json` datastructure
+    var queryResult;
+    var classitems = arrDoc.classitems;
+    for (var ici = 0; ici < classitems.length; ici++) {
+      if (classitems[ici].name === func) {
+        queryResult = classitems[ici];
+        break;
+      }
+    }
+    // bail if we couldn't find it (should never happen)
+    if (!queryResult) throw new Error('missing docs for method ' + func);
```

Comment:
I don't think this is the appropriate place to throw this error, since it's not something the user can be reasonably expected to fix. If there are no docs found, let's give validateParams() a pass somehow. If we want to check for documentation of a function we can do that elsewhere, but that's a separate issue.

Reply:
ok, i just put this in here to debug issues with missing docs. removing it won't change anything.